src/fitnesse/responders/editing/SavePropertiesResponder.java

```diff
@@ -20,20 +20,19 @@
 public class SavePropertiesResponder implements SecureResponder {
   @Override
   public Response makeResponse(FitNesseContext context, Request request) throws Exception {
-    SimpleResponse response = new SimpleResponse();
-    String resource = request.getResource();
-    WikiPagePath path = PathParser.parse(resource);
-    WikiPage page = context.getRootPage().getPageCrawler().getPage(path);
-    if (page == null)
-      return new NotFoundResponder().makeResponse(context, request);
+    PageService pageService = new PageService(context, request);
+    WikiPage page = pageService.getPage();
+    pageService.pageIsNull(page);
```

Comment:
Previously the method returned if page is null, now it just continues. That's not good.

If no test shows that this is broken now, please add such a test.

Reply:
I've checked the code and from what I've seen is that the page object can **never** be null. Otherwise the code would stop at `PathParser.parse(resource)` because of resource object being null. A page originates from a resource. For this reason I'm going to remove the null check for a page.